### PR TITLE
Added cry.0.4.0

### PR DESCRIPTION
**Pull request by toots**

### CODE REVIEW

File: packages/cry/cry.0.4.1/opam [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+homepage: "https://github.com/savonet/ocaml-cry"
+build: [
+  ["./configure" "--prefix" prefix]
+  [make]
+]
+install: [
+  [make "install"]
+]
+remove: ["ocamlfind" "remove" "cry"]
+depends: ["ocamlfind"]
```
Comment (lefessan) [inline packages/cry/cry.0.4.1/opam]: Is `ocamlfind` only needed for build (in which case you should add `{build}`), or also for its libraries (`bytes` compatibility in particular) ?

Comment (toots) [inline packages/cry/cry.0.4.1/opam]: `bytes` is needed at runtime..

**Comment by toots** [comment]
Anything else that needs action? Not sure what the tag stands for now..

**Comment by samoht** [comment]
ping @lefessan. Are you happy to merge this?

**Pull request merged**